src/main/webapp/techassistant.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>Seniors Connect</title>
+  <link rel="stylesheet" href="styles/techstyle.css">
+  <script src="script.js"></script>
+  <script src="scripts/login-script.js" defer></script>
+</head>
+<body>
+<div id="nav-placeholder"> </div>
+
+  <div class="content">
+    <div class="header">
+      <h1>Tech Assistant</h1>
+      <p>Description of Tech Assistant</p>
+    </div>
+
+    <div id="buttons">
+      <div id = "left-tutorials-button">
+        <button class="tech-button" onclick="document.location='/tutorials.html'">Tutorials</button>
+      </div>
+
+      <div id="right-forum-button">
+        <button class="tech-button" onclick="document.location='/forum.html'">Forum</button>
+      </div>
+
+      <div class="clearfix"></div>
```

Comment:
What is the clearfix for?

Reply:
Oops this was left over from some earlier styling I was trying with this page. Removed it. 
--

Reply:
Oops this was leftover from some styling I was trying earlier. Removed it now.